gravitl/netmaker
# Story/gra 1225

## Describe your changes

Added StunList and removed StunHost. StunList defaults to the server + 4 public servers (2 of ours, 2 of googles).

StunList is sent to clients (see: https://github.com/gravitl/netclient/pull/246)

## Provide Issue ticket number if applicable/not in title

GRA-1225

## Provide testing steps

1. Spin up server, use default stun list
2. add client using matching branch for client build
3. Test with a local machine and a cloud machine. Should not use proxy unless NatType is asymetric nat or double nat (can see in client logs under HOSTINFO)
4. confirm connections work as normal

## Checklist before requesting a review
